pom.xml

```diff
@@ -363,6 +363,12 @@
       <version>4.11</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+    	<groupId>org.jacoco</groupId>
+    	<artifactId>jacoco-maven-plugin</artifactId>
+    	<version>${commons.jacoco.version}</version>
+    	<scope>test</scope>
+    </dependency>
   </dependencies>
```

Comment:
Guys,
I'm not sure if we really need this, someone please comment if this should stay or be removed. Thanks,